project/publish-release.sh

```diff
@@ -106,6 +134,21 @@ echo ""
 
 git add package.json
 git commit -m "Auto-generated PR to update package.json with new version - $PACKAGE_VERSION"
-git push -f origin release-pr
+git push origin -f $currentBranch:release-pr
+
+# Make sure any pending commits on master are reset (i.e. the package.json commit)
+git reset --hard origin/$currentBranch
```

Comment:
Looking through this S/O post: http://stackoverflow.com/questions/1090309/git-undo-all-working-dir-changes-including-new-files

Sounds like is git reset --hard works (which so far it does seem to), I'm inclined to say leave as is. I'd rather leave behind a little mess and then have to figure out the right course of action then go for a nuclear option at the cost of losing some work - am I being too cautious?